core/templates/dev/head/pages/exploration-editor-page/improvements-tab/feedback-improvement-task/feedback-improvement-task.directive.ts

```diff
@@ -65,6 +65,17 @@ angular.module('oppia').directive('feedbackImprovementTask', [
             return DateTimeFormatService.getLocaleAbbreviatedDatetimeString(
               $scope.getLatestMessage().updatedOn);
           };
+
+          $scope.wasUpdatedAtMost24HoursAgo = function() {
+            var feedbackDateOrTime =
+            DateTimeFormatService.getLocaleAbbreviatedDatetimeString(
+              $scope.getLatestMessage().updatedOn);
+            if (Date.parse(feedbackDateOrTime)) {
```

Comment:
It's not recommended to use Date.parse(). See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse. Try something else instead.

Reply:
Hey @seanlip , I looked around and I feel regex could be a good option. Other options which I checked were 
- using 3rd party libs.
- `angular.isDate(new Date(String))` but `new Date(String)` is not reliable.

Should I go ahead with regex? Would like to know whats on your mind

Reply:
Don't add 3rd party libs just for this, and don't use methods/functions which are not reliable. 

Instead of parsing the string, why don't you work with the actual value of updatedOn? It should be a simple computation to find the difference between it and the current time.

Reply:
Hey, time is displayed if the date is same as today. So, technically its not **wasUpdatedAtMost24HoursAgo** but **wasUpdatedToday** . Following your advice, I used `updatedOn` to check if the update was done today.